schemas/questionnaire_v1.json

```diff
@@ -58,6 +50,16 @@
     "legal_basis": {
       "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
     },
+    "survey": {
+      "enum": ["CENSUS", "CCS"]
+    },
+    "form_type": {
+      "enum": ["H", "I", "C"]
+    },
+    "region_code": {
+      "type": "string",
+      "pattern": "^GB-[A-Z]{3}$"
```

Comment:
Given we only support a defined list of regions, this would be clearer as an enum.

Reply:
The only reason I didn't do that was because this is the same regex we use in runner's metadata parser.

Reply:
Fair enough. Documentation similar to that in the runner code would be useful here:
> An ISO 3166-2:GB region code